src/nupic/regions/TMRegion.cpp

```diff
@@ -248,17 +247,14 @@ void TMRegion::compute() {
   Output *out;
   out = getOutput("bottomUpOut");
   if (out && (out->hasOutgoingLinks() || LogItem::isDebug())) {
-    auto active = tm_->getActiveCells();         // sparse
```

Comment:
What data is output now?  It looks like it is just Active Cells at the column level?  We have another output for "activeCells" at the cell level.   So, perhaps the orColumnOutputs option is no longer needed.

But please update the description in the spec to say what this is.  TMRegion.cpp line 534 currently reads
```
  /* ----- outputs ------ */
  ns->outputs.add(
      "bottomUpOut",
      OutputSpec("The output signal generated from the bottom-up inputs "
                 "from lower levels. The width is 'numberOfCols' "
                 "* 'cellsPerColumn'.",
```

Reply:
> What data is output now? It looks like it is just Active Cells at the column level? 

yes, active as cols. Probably it should be predictiveCells as columns, but I had tests failing, so I set it to active cells to minimize changes for this PR. Followup will be #412 

> We have another output for "activeCells" at the cell level. So, perhaps the orColumnOutputs option is no longer needed. 

the option did 2 things: 
 - OR active and predictive (incorrect, removed)
 - transformed to columns -> followup "rename to asColumns option" #413 

> But please update the description in the spec to say what this is. TMRegion.cpp line 534 currently reads 

good, I knew it's somewhere, but couldn't find it. thanks :+1: